src/components/checkboxes/checkboxes.yaml

```diff
@@ -241,6 +241,22 @@ examples:
         hint:
```

Comment:
_If_ we're allowing describedBy to be passed as a 'root' option for the checkboxes component, we should document it in `params`.

Reply:
(Having reviewed the rest of the PR, I suspect we're not so this is probably irrelevant)

Reply:
So this relates to https://github.com/alphagov/govuk-frontend/pull/1347#discussion_r285124421, but I forgot to add the root param.

When used without a fieldset, this is where `describedBy` can be set.

I'd make this the description?

>One or more element IDs to add to the input `aria-describedby` attribute without a fieldset, used to provide additional descriptive information for screenreader users.

Unless you'd prefer not to allow `describedBy` to be customised when no fieldset is set?

Reply:
I'll resolve this one as we're chatting on https://github.com/alphagov/govuk-frontend/pull/1347#discussion_r286029798